index.d.ts

```diff
@@ -1167,6 +1214,45 @@ declare module '@amaas/amaas-core-sdk-js' {
       apiVersion?: string
       token?: string
     }): void
+    namespace PositionPNL {
+      function retrieve(
+        { AMId }: { AMId: number },
+        bookId: string,
+        businessDate: Date,
+        callback?: Function
```

Comment:
You'll need to update this to ensure that it matches the source. Currently you are exporting the transaction PnL functions under a `transactionPnL` module, so in this file you will need to make sure that it looks something like:

```typescript
namespace api {
  // ...
  namespace transactionPnL {

    function retrieve(...)
    function insert(...)
    // ...

  }
}
```
Otherwise, consumers of the SDK in a TS environment will face a compiler error when they attempt to
```typescript
import { api } from '@amaas/amaas-core-sdk-js'

api.transactionPnL.retrieve() // TS Error: no such function
```